Clean up debug feature (#615)
SFDO-Tooling/Snowfakery: docs/index.md

### Debug

> Snowfakery's `debug` function can be used to output values to your command line for debugging. Specifically, it outputs the value to `stderr`.

Reviewer: ```suggestion
You can use Snowfakery's `debug` function to output values to `stderr` (usually, the command line) for debugging. 
```
